templates/news/news-en.html

```diff
@@ -3,224 +3,476 @@
 	"title": "News",
 	"language": "en",
 	"altLangPage": "news-fr.html",
-	"breadcrumbs": false,
 	"secondlevel": false,
-	"dateModified": "2023-09-01",
+	"dateModified": "2024-09-13",
 	"share": "true",
 	"pageType": "news"
 }
 ---
-<div class="mrgn-bttm-md" data-wb-randomize='{ "shuffle": true, "selector": "a", "toggle": "hide" }'>
-	<a href="#" class="hide">
+
+<!-- START: Banner (This pattern should be reviewed, optimized and changed to incldue stretched-link) -->
+<div class="mrgn-bttm-md" data-wb-randomize="{ &quot;shuffle&quot;: true, &quot;selector&quot;: &quot;a&quot;, &quot;toggle&quot;: &quot;hide&quot; }">
+	<h2 class="wb-inv">Promotions</h2>
+	<a href="https://www.international.gc.ca/world-monde/issues_development-enjeux_developpement/response_conflict-reponse_conflits/crisis-crises/israel.aspx?lang=eng" class="promo hide learmore" data-gc-analytics="promo:homePageFeature">
 		<figure>
-			<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/F5F5F5/000000.png" alt="">
+			<img class="img-responsive full-width" src="https://dummyimage.com/1170x347/F5F5F5/000000.png" alt="">
 			<figcaption class="well brdr-rds-0">
-				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
+				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Canada’s response to the crisis in Israel, the West Bank and the Gaza Strip</span>
 			</figcaption>
 		</figure>
 	</a>
-	<a href="#" class="hide">
-		<figure><img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/26374A/FFFFFF.png" alt="">
+	<a href="https://www.international.gc.ca/world-monde/issues_development-enjeux_developpement/response_conflict-reponse_conflits/crisis-crises/ukraine.aspx?lang=eng&amp;utm_campaign=gac-amc-ukraine-21-22&amp;utm_source=canadahome&amp;utm_medium=feat&amp;utm_content=en" class="promo hide learmore" data-gc-analytics="promo:homePageFeature">
```

Comment:
We should avoid to put real URL in our template. If we want to demo a test url, we can use https://example.com or https://example.org which is designed for that purpose.